assets/js/playlist.js

```diff
@@ -10,10 +10,15 @@
       url: $(this).attr('action'),
       data: data
     }).done(function(response){
+      // add videoID to player
+      song_ids.push(response.link);
+      // create the necessary csrftoken for the delete song form
+      var csrftoken = getCookie('csrftoken');
```

Comment:
remove this, csrf_token, token already pass from your form.

Reply:
Updated.